avail/src/main/kotlin/avail/compiler/splitter/MessageSplitterTokenizer.kt

```diff
@@ -87,7 +88,7 @@ class MessageSplitterTokenizer
 	private val messageNameSize: Int = messageName.tupleSize
 
 	/** The current one-based index into the messageName. */
-	private var positionInName = 1
+	private var startInName = 1
```

Comment:
The field name has changed which implies its usage/purpose has changed, however the comment has not been updated. Is the comment still accurate?